packages/sqip/src/sqip.js

```diff
@@ -110,17 +226,17 @@ async function processImage({ filePath, config }) {
       sqipConfig: config,
       pluginOptions,
       metadata,
-      filePath
+      // filePath
```

Comment:
It seems to work with this line commented out 🤷‍♂. I have to check why it was needed (maybe for the CLI?)

Reply:
Some plugins require the file to exist on the file system.

We should change these plugins to create a temporary file if they do not support buffers as input